src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -744,14 +744,31 @@ internal static bool IsPackageReferenceProject(Project project)
                     if (matchingPackages.Any())
                     {
                         var topLevelPackage = matchingPackages.Single();
-                        InstalledPackageReference installedPackage;
+                        InstalledPackageReference installedPackage = default;
 
                         //If the package is not auto-referenced, get the version from the project file. Otherwise fall back on the assets file
                         if (!topLevelPackage.AutoReferenced)
                         {
                             try
                             { // In case proj and assets file are not in sync and some refs were deleted
-                                installedPackage = projPackages.Where(p => p.Name.Equals(topLevelPackage.Name, StringComparison.Ordinal)).First();
+                                if (assetsFile.PackageSpec.RestoreMetadata.CentralPackageVersionsEnabled)
+                                {
+                                    foreach (KeyValuePair<string, CentralPackageVersion> packageCentralVersion in tfmInformation.CentralPackageVersions)
+                                    {
+                                        if (packageCentralVersion.Key.Equals(topLevelPackage.Name, StringComparison.Ordinal))
+                                        {
+                                            installedPackage = new InstalledPackageReference(topLevelPackage.Name)
+                                            {
+                                                OriginalRequestedVersion = topLevelPackage.VersionOverride?.MinVersion.ToString() ?? packageCentralVersion.Value.VersionRange.MinVersion.ToString(),
```

Comment:
The rest of the code handles that by displaying the range. I'm guessing original version. We should follow that pattern.

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net8.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="NuGet.Common" Version="[,6.6.1]" />
    <PackageReference Include="NuGet.Commands" Version="6.5.0" />
    <PackageReference Include="NuGet.Protocol" Version="[6.5.0,)" />
  </ItemGroup>
</Project>

```

```console
PS C:\Code\Temp\DLP> dotnet list package 
Project 'DLP' has the following package references
   [net8.0]: 
   Top-level Package      Requested   Resolved
   > NuGet.Commands       6.5.0       6.5.0
   > NuGet.Common         [,6.6.1]    3.2.0
   > NuGet.Protocol       [6.5.0,)    6.5.0

```

Reply:
You're right, I was under the impression that Ranges weren't allowed, floating are not allowed. 

I'll have to investigate this more, I was using `MinVersion` because `VersionRange.OriginalString` is always a range even for if you're not using one.